resources/views/students/about.blade.php

```diff
@@ -22,11 +22,11 @@
 <div class="container">
     <h1><i class="fas fa-rocket"></i> Get Started with Student Research</h1>
 
-    <div class="card student-info-card border-0 bg-dark text-white">
+    <div class="card student-info-card border-0 bg-dark text-white" style="z-index: 0;">
```

Comment:
Since you're already creating a sass file `_student.scss`, then this z-index statement would make more sense there instead of inline, i.e.

```css
.student-info-card {
    z-index: 0;
}
```

Reply:
The changes requested have been applied and committed.